### DISCUSSION TITLE
目录和文件的排序应该怎么设置?

### DISCUSSION THREAD

**Discussion by Zephery** [vuepress-theme-hope/vuepress-theme-hope#3891] [Q&A] [answered]
<img width="327" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/14806207/8017483a-ff95-4ece-ad38-fa6f8d36e7ef">

请问怎么统一把这里的目录放到最后面？

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/6fe8070d-90df-47a7-ade5-4fad5afe4c9a)

是文档哪里说的不清楚？

```md
---
dir:
  order: xxx
---

<!-- dir/README.md -->
```

**Reply by Zephery** [reply]
设置了，没有生效
https://github.com/Zephery/MyWebsite/commit/b004274513f96953af13863120534cd7f841b58a

**Reply by Mister-Hope** [reply]
A reproduction link plz? I can not find a link using this.

It doesn't seems like you have a valid sidebar that is sorting java folder:
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/33315834/9b1370f2-c705-4a63-bcbe-0f7b2b940e31)

**Reply by Zephery** [reply]
Hi~
https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/theme.ts
https://github.com/Zephery/MyWebsite/blob/master/docs/.vuepress/config.ts
